alacritty/src/renderer/mod.rs

```diff
@@ -1,33 +1,26 @@
-use std::collections::HashMap;
-use std::hash::BuildHasherDefault;
-use std::mem::size_of;
-use std::{fmt, ptr};
+use std::ffi::CStr;
+use std::fmt;
 
-use bitflags::bitflags;
-use crossfont::{
-    BitmapBuffer, Error as RasterizerError, FontDesc, FontKey, GlyphKey, Metrics, Rasterize,
-    RasterizedGlyph, Rasterizer, Size, Slant, Style, Weight,
-};
-use fnv::FnvHasher;
-use log::{error, info};
-use unicode_width::UnicodeWidthChar;
+use crossfont::Metrics;
+use log::info;
 
 use alacritty_terminal::index::Point;
 use alacritty_terminal::term::cell::Flags;
 use alacritty_terminal::term::color::Rgb;
 use alacritty_terminal::term::SizeInfo;
 
-use crate::config::font::{Font, FontDescription};
-use crate::config::ui_config::{Delta, UiConfig};
 use crate::display::content::RenderableCell;
 use crate::gl;
-use crate::gl::types::*;
 use crate::renderer::rects::{RectRenderer, RenderRect};
-use crate::renderer::shader::{ShaderError, ShaderProgram};
+use crate::renderer::shader::ShaderError;
 
-pub mod builtin_font;
 pub mod rects;
 mod shader;
+mod text;
+
+use shader::ShaderVersion;
+use text::Glsl3Renderer;
+pub use text::{Gles2Renderer, GlyphCache, LoaderApi, TextRenderer};
```

Comment:
```suggestion
pub use text::{Gles2Renderer, GlyphCache, LoaderApi, TextRenderer};
use shader::ShaderVersion;
use text::Glsl3Renderer;
```

Not sure if rustfmt did this, but `pub` should usually come first.